xcm/src/v0/traits.rs

```diff
@@ -125,6 +125,8 @@ impl Outcome {
 	}
 }
 
+/// Something that can execute an xcm message.
+// TODO: why is this both generic and associated?
```

Comment:
open question still

Reply:
You responded to this in another PR. Will remove my todo as you'd remove the associated type there.